### PR TITLE
[#173792786] Disable auth level SpidL1

### PR DESCRIPTION
**Pull request by BurnedMarshal**
Disable SpidL1 and validate authLevel query param

### CODE REVIEW

**Comment by gunzip** [comment]
this will block spid testenv (and google / apple accounts) from working cc @francescoparisidev @pp-ps @cloudify

File: src/config.ts [hunk 1]
```diff
@@ -44,6 +44,8 @@ export const SPID_LEVELS = {
   SpidL3: "https://www.spid.gov.it/SpidL3"
 };
 
+export const ENABLED_SPID_LEVELS: ReadonlyArray<string> = ["SpidL2", "SpidL3"];
```
Comment (gunzip) [inline src/config.ts:47]: this must be a configuration that comes from io-backend someway

File: docker-compose.yml [hunk 2]
```diff
@@ -40,12 +40,12 @@ services:
     networks:
       - io-spid-commons
 
-  spid-saml-check:
-    image: spid-saml-check
-    ports:
-      - "8080:8080"
-    networks:
-      - io-spid-commons
+  #spid-saml-check:
```
Comment (gunzip) [inline docker-compose.yml:43]: why this change?

Comment (BurnedMarshal) [reply docker-compose.yml:43]: `spid-saml-check` is no more available on docker-hub. Can we use [this](https://hub.docker.com/r/superdell/spid-saml-check) container?

Comment (gunzip) [reply docker-compose.yml:43]: nope, it's better to leave this as-is and bake our own locally from github source code 
(we should add such a docker image into this repo probably)

File: src/config.ts (hunk 1 above)
Comment (BurnedMarshal) [reply src/config.ts]: Fixed in 92a5f1f

File: src/index.ts [hunk 3]
```diff
@@ -227,7 +231,30 @@ export function withSpid({
...
+            .chain(
+              fromPredicate(authLevel =>
+                appConfig.spidLevelsWhitelist.includes(authLevel)
+              )
+            )
+            .foldL(
+              () =>
+                ResponseErrorValidation(
+                  "Bad Request",
+                  "Missing or invalid authLevel"
```
Comment (gunzip) [inline src/index.ts:251]: can we log `q.authLevel` here?

Comment (BurnedMarshal) [reply src/index.ts:254]: Fixed in 46a11f6
